backend/utilities/cron.js

```diff
@@ -0,0 +1,39 @@
+import { CronJob } from 'cron';
+import fs from 'fs';
+
+import Movie from '../models/Movie.js';
+
+const removeMovie = async (movie) => {
+  const subtitlesPath = `./subtitles/${movie.imdbCode}`;
+  // TO DO check serverLocation is working or not
```

Comment:
I think this bug exists because the subtitle route is run way faster then the movie route can initialise in some cases.